src/server/game/Maps/Map.h

```diff
@@ -155,6 +159,16 @@ struct LiquidData
     float  depth_level;
 };
 
+struct PositionFullTerrainStatus
+{
+    typedef VMAP::AreaAndLiquidData::AreaInfo AreaInfo;
```

Comment:
I know this is convenient but don't do this. Don't make all users of Map class also include VMapManager